webpack/assets/javascripts/react_app/components/Editor/EditorHelpers.js

```diff
@@ -0,0 +1,157 @@
+import {
+  isEmpty,
+  filter,
+  find,
+  replace,
+  capitalize,
+  forEach,
+  concat,
+  sortBy,
+} from 'lodash';
+
+import 'ace-builds/src-noconflict/ace';
```

Comment:
Not sure if it's "legal" from the packaging perspective :/ But without this the tests are failing, although everything just works (didn't test on production though :/)

Reply:
I think this is not ok, as that is not exported in https://github.com/theforeman/foreman-js/pull/363/files#diff-49c3b2f1af7619149c2b3f68c0fd26f135c1dcc02932cf76f510da25c69bebc5R65 tho I'm definitelly not sure. Maybe @MariaAga would know better?

Reply:
Maybe this is the best solution :thinking: 
and then we can add it to `foreman-js`

Reply:
Yup, tho we will need to package new version of `vendor-js`